cura/PlatformPhysics.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, controller, volume):
 
         Application.getInstance().getPreferences().addPreference("physics/automatic_push_free", False)
         Application.getInstance().getPreferences().addPreference("physics/automatic_drop_down", False)
+        Application.getInstance().getPreferences().addPreference("physics/per_model_drop", False)
```

Comment:
I don't think we should make this temporary variable a preference -- those are basically only used if it's a user setting the program needs to remember between sessions (so when the program closes). This seems like both overkill and hacking the preference system.

Since we're in the same session, there must be another way to properly communicate this to the platform physics object. Maybe the platform physics could pull it from somewhere, but since they're separate systems, perhaps a signal (like 'suppress drop to buildplate only for the next operation') would be better.